lib/vdsm/api/vdsm-api.yml

```diff
@@ -9752,6 +9752,11 @@ ManagedVolume.attach_volume:
     -   description: The information needed to attach a volume
         name: connection_info
         type: *ManagedVolumeConnection
+    -   defaultvalue: null
```

Comment:
Using blank uuid (00000000-0000-0000-0000-000000000000) in the schema will simplify all the APIs in the entire system. Regardless of engine version, we always have a sd_id and we never have to handle the case when sd_id=None.